wp-content/themes/core/pcss/content/panel/_hero.pcss

```diff
@@ -19,7 +55,60 @@
 		z-index: 1;
 	}
 
-	@media (--viewport-medium) {
-		height: 100vh;
+	/* CASE: Split Right/Left Layout */
+	.hero--layout-content-split-left &,
+	.hero--layout-content-split-right & {
+		position: relative;
+
+		&:before {
+			content: '';
+			position: relative;
+			padding-bottom: 85%;
+			display: block;
+			background: transparent;
+		}
+
+		@media (--viewport-medium) {
+			position: absolute;
+			width: 40%;
+
+			&:before {
+				display: none;
+			}
+		}
+	}
+
+	/* CASE: Split Right Layout */
+	.hero--layout-content-split-right & {
+		@media (--viewport-medium) {
+			left: auto;
+			right: 0;
+		}
+	}
+}
+
+.hero__content {
+	padding-top: var(--spacer-6);
+	padding-bottom: var(--spacer-6);
+
+	/* CASE: Split Right/Left Layout */
+	.hero--layout-content-split-left &,
```

Comment:
I can see this being a layout we do fairly frequently for a project and across panels. Would you be down to create a layout helper class that accomplishes the split layout (see `/base/layouts/`)? 😄